src/components/Chat/RoomMenu.tsx

```diff
@@ -6,7 +6,7 @@ import "./RoomMenu.scss";
 import clsx from "clsx";
 import date from "date-and-time";
 import { Base64 } from "js-base64";
-import React, { ReactNode, useContext, useMemo, useRef, useState } from "react";
```

Comment:
this can all be cleaned up once the other stuff is removed... i don't think this file needs to change at all